### ISSUE TITLE
Math doesn't display on some docs pages

### ISSUE THREAD

**Issue by gdalle** [TuringLang/Turing.jl#1827]
This is the case for instance with https://turing.ml/dev/docs/for-developers/variational_inference
I'll try to submit a PR

**Comment by devmotion** [comment]
Probably relevant: https://github.com/TuringLang/turing.ml/issues/8

**Comment by tomroesch** [comment]
Before #1817 the website looked a lot worse, but there is still work to do. I'll see if I can fix the remaining issues using the mentioned post, it looks very relevant.

**Comment by filchristou** [comment]
is there a temporary way to read through this page somehow, e.g. compiling it locally ?

**Comment by tomroesch** [comment]
There are instructions to build the website locally [here](https://github.com/TuringLang/Turing.jl/tree/master/docs).  However, there were some issues for me, which I explained with possible fixes in #1855.

I guess to test this PR one could clone the [repository](https://github.com/tomroesch/Turing.jl) I requested the PR from and build that locally (it also includes the fixes to build the website locally).
